package.json

```diff
@@ -15,7 +16,7 @@
     "android": "react-native-scripts android",
     "ios": "react-native-scripts ios",
     "test": "node node_modules/jest/bin/jest.js --watch",
-    "postinstall": "remotedev-debugger --hostname localhost --port 8081 --injectserver"
+    "debugger": "remotedev-debugger --hostname localhost --port 8081 --injectserver"
```

Comment:
i changed this from postinstall to debugger (so you would run `npm run debugger`) because it's possible (eg: in my case) that the user doesn't have remotedev-debugger on their machine